### PR TITLE
Feat: useDocumentTitle, add data for ecom, fix redirects

### PR DESCRIPTION
**Pull request by sudipkundu999**
### In this PR,
- I'm adding data of products to be used in e-commerce
- Fixing page not found error by adding a `_redirects` file
- Adding a custom hook [useDocumentTitle](https://github.com/sudipkundu999/CobraStore-React/pull/4/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&file-filters%5B%5D=No+extension&show-deleted-files=true&show-viewed-files=true#diff-4b50d0125785fd3a19b46bc19787fd2319449b8a94b31c75b53718961cb746bd)

### Please review these files only! [Click here](https://github.com/sudipkundu999/CobraStore-React/pull/4/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&file-filters%5B%5D=No+extension&show-deleted-files=true&show-viewed-files=true)

### CODE REVIEW

File: src/components/ProductCard.jsx [hunk 1]
```diff
@@ -1,19 +1,13 @@
-export const ProductCard = ({ props }) => {
```
Comment (zoef-yt) [inline src/components/ProductCard.jsx:1]: UI suggestion: add some spacing inside the card 
image

**Review by zoef-yt — APPROVED** [review]
The code looks good. Great that you are using js docs for function, The UI looks good too just do the suggested changes. Good to merge. good work sudip.

File: src/components/ProductCard.jsx (hunk 1 above)
Comment (sudipkundu999) [reply src/components/ProductCard.jsx:1]: Done

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**